docs/book/src/user/quick-start.md

```diff
@@ -731,7 +731,19 @@ spec:
 {{#/tab }}
 {{#/tabs }}
 
-After the controlplane is up and running, let's retrieve the [target cluster] Kubeconfig:
+To verify the control plane is up, check if the control plane machine
```

Comment:
I'm verifying that the control plane is up and not necessarily running. That happens below with the `get nodes` command.